tests/wpt/meta/css/css-align/blocks/align-content-block-002.html.ini

```diff
@@ -30,19 +30,19 @@
     expected: FAIL
 
   [.test 1: start]
-    expected: FAIL
+    expected: PASS
 
   [.test 4: baseline]
-    expected: FAIL
+    expected: PASS
```

Comment:
@rayguo17 Blocks like this should also be removed rather than changed to `PASS`. You can just delete the:

```ini
  [.test 4: baseline]
    expected: FAIL
```

Entirely. And same for other newly-passing subtests.